README.md

```diff
@@ -243,26 +243,34 @@ Go to the [CLI README](https://github.com/raiden-network/light-client/tree/maste
 
 ### Install and Run the dApp
 
-1. **Clone repository**  
+1. **Clone repository**
 
    ```bash
    git clone --recurse-submodules https://github.com/raiden-network/light-client.git
    cd light-client
    ```
 
-2. **Install dependencies**  
+2. **Install dependencies**
 
    ```bash
    yarn install
    ```
 
-3. **Build the Raiden SDK**  
+3. **Update the sdk submodules**
+
+   ```bash
```

Comment:
Adding a `update-submodules` script to the raiden-ts [`package.json`](https://github.com/raiden-network/light-client/blob/master/raiden-ts/package.json) would allow you to run the command like so without switching directories:

```bash
yarn workspace raiden-ts update-submodules
```